clients/multiversx/client.go

```diff
@@ -47,6 +50,8 @@ type ClientArgs struct {
 	RoleProvider                 roleProvider
 	StatusHandler                bridgeCore.StatusHandler
 	ClientAvailabilityAllowDelta uint64
+	eventsBlockRangeFrom         int64
```

Comment:
Why are these unexported? How can we set them outside of this package? For example, if a factory will be defined somewhere else, these can not be set as they are private to the multiversx package